### DISCUSSION TITLE
Bump on previous post

### DISCUSSION THREAD

**Discussion by james-a-morris** [ethereum-optimism/developers#314] [Application Development]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I was hoping for the team to revisit this: https://github.com/ethereum-optimism/developers/discussions/309#discussioncomment-8884104

I am following the guidance of a user from the `#dev-support` thread I made in the OP Discord. (img attached) 
![image](https://github.com/ethereum-optimism/developers/assets/96435344/b47ca0a6-4168-421b-bba7-b870c111d33d)

--

The primary follow-up is that the most recent version of `@eth-optimism/sdk` contains [`@eth-optimism/bedrock-contracts@0.17.1`](https://www.npmjs.com/package/@eth-optimism/contracts-bedrock/v/0.17.1?activeTab=code) which still has the faulty `preinstall` script:
```
    "preinstall": "npx only-allow pnpm",
```

It has been removed from the main branch on Git, but a new package hasn't been published. I think the steps needed to solve are:
1. Bump the `@eth-optimism/bedrock-contracts` package to a higher version on NPMJS which includes removing the `preinstall` script in `package.json`
2. Bump the `eth-optimism/sdk` package to a higher version on NPMJS that contains the bumped package from step 1. 

If needed I can provide failing build information that highlights the issue we're having. I'd prefer to post this privately, however.

### Additional Information

cc @evaldofelipe

**Comment by sbvegan** [comment]
I bumped this internally and here's the PR that addresses your request: https://github.com/ethereum-optimism/optimism/pull/9964

**Comment by roninjin10** [comment]
Try sdk 3.3.0 which has this fix
